polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -278,6 +278,9 @@ pub mod pallet {
 		/// parachain.
 		type DefaultChannelSizeAndCapacityWithSystem: Get<(u32, u32)>;
 
+		/// Means of converting an `Xcm` into a `VersionedXcm`.
```

Comment:
A little bit more docs would be nice. For example that `pallet-xcm` implements this for example.

Reply:
thank you, updated: https://github.com/paritytech/polkadot-sdk/pull/4281/commits/d8637569808eb3881cfa8aba9c919584acf98644